CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+## Style
+
+Archivematica uses the Python [PEP8](https://www.python.org/dev/peps/pep-0008/) community style guidelines.
+Newly-written code should conform to PEP-8 style.
+PEP8 is a daunting document, but there are very good linters available that you can run to check style in your code.
+The Archivematica development uses [Sublime Text](sublimetext.com/) with the [SublimeLinter-flake8](https://github.com/SublimeLinter/SublimeLinter-flake8) plugin, which automatically scans your code and displays any style notes inline.
+Similar plugins are available for other editors, or the [flake8](https://pypi.python.org/pypi/flake8) tool can be used at the command line.
+
+### Some extra notes
+
+A few additional stylistic preferences might not get flagged by linters:
+
+* Don't use variable or parameter names that shadow builtin functions and types.
+  For example, don't name a variable "file".
+  (Unfortunately, Python uses many useful names for its builtin types and functions.)
+* Most of the time, prefer importing specific names from modules instead of importing the entire module.
+  This makes it easier to tell at the top of a file what is being used from its dependencies.
+  For example:
+
+```python
+# Good
+
+from main.models import File
+
+f = File.objects.get(uuid="something")
+
+# Bad
+
+import main.models
+
+f = models.File.objects.get(uuid="something")
```

Comment:
Interesting - I prefer the opposite. I'd rather know when I'm looking at a line if it's from another module (especially stdlib) or defined in this file.  It also makes it easier to use more things from the same module.

I care less for Django models though, probably because you're more likely to import multiple models.py and doing ```from main import models; from fpr import models as fpr_models``` is a pain.